src/js/components/Tag/Tag.js

```diff
@@ -1,7 +1,6 @@
 import React, { forwardRef, useContext } from 'react';
 import { ThemeContext } from 'styled-components';
-
-import { FormClose } from 'grommet-icons/icons/FormClose';
+import { FormClose } from 'grommet-icons';
```

Comment:
```suggestion
import { FormClose } from 'grommet-icons/icons/FormClose';
```

Reply:
@vinodkv2511 should fix the error..

Reply:
Updated!

Reply:
![image](https://github.com/grommet/grommet/assets/44632502/4573a77c-893f-434c-ba89-40e77a37a0a1)


This time the bundle size is 0.1KB more than original. @britt6612

Reply:
@jcfilben  @britt6612 any way to fix this? Or is there a way to increase the bundle size?

Reply:
Hey @vinodkv2511 in the `package.json` file you can set the `bundlesize.maxSize` property to "162 kB"

Reply:
Thanks @jcfilben. I have update it to 162KB.